test/gas/custom_gas_price.js

```diff
@@ -0,0 +1,22 @@
+const assert = require("assert");
+const getWeb3 = require("../helpers/web3/getWeb3");
+
+describe("Custom Gas Price", () => {
+  it("should return gas price of 15 when specified as a decimal", async() => {
```

Comment:
`it("should return gas price of 15 when specified as a decimal")` -- this doesn't make any sense to me - whenever I specify gas price as a decimal it should come back as 15 regardless of what the value is?

Reply:
You are correct and both tests in this file backs the assumptions. Whether you specify `15` or `0xf` for `gasPrice`, `getGasPrice()` successfully returns the decimal value of `15`.

Reply:
Added new tests.